components/adv-post-form.js

```diff
@@ -17,14 +17,34 @@ export function AdvPostInitial ({ forward, boost }) {
   }
 }
 
-export default function AdvPostForm () {
+export default function AdvPostForm ({ edit, isLink }) {
   const me = useMe()
 
   return (
     <AccordianItem
       header={<div style={{ fontWeight: 'bold', fontSize: '92%' }}>options</div>}
       body={
         <>
+          {!edit && isLink &&
+            <MarkdownInput
+              label={
+                <div className='d-flex align-items-center'>comment
+                  <Info>
+                    <span className='fw-bold'>Use this initial comment to do e.g. any of the following:</span>
+                    <ul>
+                      <li>Provide context</li>
+                      <li>Summarize the link</li>
+                      <li>Note the reason why you are posting the link</li>
+                      <li>Respect readers time by helping them decide whether to click the link or not</li>
+                      <li>Spur great conversation!</li>
+                    </ul>
+                  </Info>
```

Comment:
I would change the info text to this:

> You can use this initial comment to ...
> - provide context
> - summarize the link
> - spur great conversation!
> - or for anything else that comes to your mind

Reasons:

- I don't like long text where shorter text can convey the same in a more efficient manner  - I assume users prefer "efficient text", too
- "Note the reason why you are posting the link" feels too similar to "provide context".
- "Respect readers time ..." is imo setting too much of a tone - I don't know how to explain it better, lol